### PR TITLE
fix: countdown label visibility

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Remove the label when there is no active countdown.

# Changes

- expose countdown visible state
- limit label visibility

# Tests

Tested manually
| Before | After |
|--------|--------|
| image | image |

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte [hunk 1]
```diff
@@ -36,11 +38,16 @@
...
+        {#if isCountdownVisible}
+          <span class="caption description"
+            >{$i18n.proposal_detail__vote.expiration}</span
+          >
+        {/if}
         <div class="caption value">
-          <Countdown {deadlineTimestampSeconds} />
+          <Countdown
+            {deadlineTimestampSeconds}
+            bind:visible={isCountdownVisible}
```
Comment (lmuntaner) [inline frontend/src/lib/components/proposal-detail/VotesResults.svelte:49]: Wouldn't it be better to not rely on the value inside here, and set it at this level instead?

```
$: isCountdownVIsible = deadlineTimestampSeconds > nowInSeconds()
```

I'm guessing that's the logic...

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte
Comment (lmuntaner) [inline frontend/src/lib/components/proposal-detail/VotesResults.svelte:1]: Can you add a test for this also?

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/VotesResults.svelte]: Good point! Updated.

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/VotesResults.svelte:1]: Sure ✔️

**Review by lmuntaner — APPROVED** [review]
Thanks!

**Pull request merged**